docs_headless/src/content/docs/reference/example.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Example Reference
+description: A reference page in my new Starlight docs site.
+---
+
+Reference pages are ideal for outlining how things work in terse and clear terms.
+Less concerned with telling a story or addressing a specific use case, they should give a comprehensive outline of what you're documenting.
+
+## Tests
+
+- [General Concepts](/guides/architecture)
+- [Data Fetching](/guides/datafetchingguide)
```

Comment:
It's too bad we can't use links to the .md files, because it means the md docs won't be browsable from Github directly. 
Can you try something like https://github.com/vernak2539/astro-rehype-relative-markdown-links?